libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -249,6 +249,21 @@
 #define HAL_BARO_MS5611_I2C_BUS 1
 #define HAL_BARO_MS5611_I2C_ADDR 0x77
 #define HAL_COMPASS_DEFAULT HAL_COMPASS_HMC5843_MPU6000
+#define HAL_OPTFLOW_ONBOARD_VDEV_PATH "/dev/video0"
+#define HAL_OPTFLOW_ONBOARD_SENSOR_WIDTH 320
+#define HAL_OPTFLOW_ONBOARD_SENSOR_HEIGHT 240
+#define HAL_OPTFLOW_ONBOARD_OUTPUT_WIDTH 64
+#define HAL_OPTFLOW_ONBOARD_OUTPUT_HEIGHT 64
+#define HAL_OPTFLOW_ONBOARD_CROP_WIDTH 240
+#define HAL_OPTFLOW_ONBOARD_CROP_HEIGHT 240
+#define HAL_OPTFLOW_ONBOARD_NBUFS 8;
+#define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
+#define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
+#define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_PARAM_DEFAULTS_PATH "/etc/arducopter/minlure.parm"
+/* focal length 3.6 um, 2x binning in each direction
+ * 240x240 crop rescaled to 64x64 */
+#define HAL_FLOW_PX4_FOCAL_LENGTH_MILLIPX (2.5 / (3.6 * 2.0 * 240 / 64))
```

Comment:
I would be surprised if you happened to have the exact same focal length, pixel size, binning characteristics as the Bebop here.